src/rabbit_queue_location_client_local.erl

```diff
@@ -0,0 +1,47 @@
+%%% The contents of this file are subject to the Mozilla Public License
+%%% Version 1.1 (the "License"); you may not use this file except in
+%%% compliance with the License. You may obtain a copy of the License at
+%%% http://www.mozilla.org/MPL/
+%%%
+%%% Software distributed under the License is distributed on an "AS IS"
+%%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+%%% License for the specific language governing rights and limitations
+%%% under the License.
+%%%
+%%% The Original Code is RabbitMQ.
+%%%
+%%% @author Ayanda Dube <ayanda.dube@erlang-solutions.com>
```

Comment:
we don't use author tags in our code, nor the _Created_ one bellow, or the `@doc` one like bellow. We do have doc headers, see `gm.erl` for an example.

Author and Date are stored on git anyway.